### PR TITLE
Feature/tab navigation interviews and jobs section

### PR DESCRIPTION
**Pull request by mariusgolabski**
- #8

### CODE REVIEW

File: components/Job/Job.Styled.js [hunk 1]
```diff
@@ -0,0 +1,100 @@
+import { styled } from "styled-components";
```
Comment (Kheirah) [inline components/Job/Job.Styled.js:1]:
```suggestion
import styled from "styled-components";
```

File: components/TabButton/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  return (
+    <StyledButton
+      $isActive={isActive}
+      onClick={() => {
+        onClick();
+      }}
+    >
+      {children}
+    </StyledButton>
+  );
```
Comment (Kheirah) [inline components/TabButton/index.js:16]: If I'm not mistaken about your intent, this is the same but shorter.
```suggestion
  return (
    <StyledButton
      $isActive={isActive}
      onClick={onClick ? onClick : null}
    >
      {children}
    </StyledButton>
  );
```

**Review by Kheirah — APPROVED** [review]
Good job!

**Comment by mariusgolabski** [comment]
Hi, thanks for your review! - the changes are now implemented.

**Comment by mariusgolabski** [comment]
Hi, thanks for your review! - the changes are now implemented.

**Comment by mariusgolabski** [comment]
Hi, thanks for your review! - the changes are now implemented.

**Pull request merged**